liberty-maven-plugin/src/it/dev-it/src/test/java/net/wasdev/wlp/test/dev/it/MultiModuleUpdatePomsTest.java

```diff
@@ -111,6 +120,10 @@ public void updatePomsTest() throws Exception {
             countOccurrences("guide-maven-multimodules-war tests compilation was successful.", logFile));
       assertEquals(getLogTail(), ++earTestsCount,
             countOccurrences("guide-maven-multimodules-ear tests compilation was successful.", logFile));
+
+      // verify that feature generation did not run since there are no source class
```

Comment:
The only comment I have is that I don't like that the check for "feature gen regenerated the same features" and "feature generation did not run" is the same. Those are two unique scenarios and we should have more specific checks in place.

It's okay for now, but I think this is something we should update in the future.

Reply:
Good point, something like detecting the 'regenerate' message.